production/helm/loki-stack/requirements.yaml

```diff
@@ -7,4 +7,11 @@ dependencies:
   condition: promtail.enabled
   repository: "file://../promtail"
   version: "^0.6.0"
-
+- name: "grafana"
+  condition: grafana.enabled
+  version: "~3.3.7"
+  repository:  "@stable"
```

Comment:
Should use the full URL in case people have removed this repo locally. `https://kubernetes-charts.storage.googleapis.com/`